### ISSUE TITLE
PyOxidizerFinder.get_data

### ISSUE THREAD

**Issue by jayvdb** [indygreg/PyOxidizer#139]
[`ResourceLoader.get_data`](https://docs.python.org/3/library/importlib.html#importlib.abc.ResourceLoader.get_data) is deprecated, but it is used by nose, which isnt likely to be updated any time soon.

```py
Traceback (most recent call last):
  File "nose.core", line 252, in usage
AttributeError: 'PyOxidizerFinder' object has no attribute 'get_data'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "nose.core", line 121, in __init__
  File "unittest.main", line 100, in __init__
  File "nose.core", line 145, in parseArgs
  File "nose.core", line 256, in usage
NameError: name '__file__' is not defined
>>> nose.__loader__
<PyOxidizerFinder object at 0x7f1f79492810>
```

According to the latest v3 docs, this is used by [`pkgutil.get_data`](https://docs.python.org/3/library/pkgutil.html#pkgutil.get_data) which is very broadly used.

A quick look in my site-packages indicate it is also used by `pkg_resources` (vendored by pip, pipenv, etc), `dateutil`, `werkzeug`, `jsonschema`, `virtualenv`, `pydocstyle`, `text_unidecode`, `metakernel` (base of several ipython kernels), `selenium`, `plotly`, `scrapy`, `pyramid`, `dash`, & `hexdump` (I excluded many others that are less commonly used)

Optionally used by `importlib_resources`

**Comment by indygreg** [comment]
Yeah, we should probably implement this to help ease porting to `ResourceReader`.

**Comment by indygreg** [comment]
`get_data()` is implemented as of 86bba2046b15102a5808ad51bd49c4c81d5419f6.
